backend/repository/handler.go

```diff
@@ -361,6 +362,10 @@ func (h *Handler) CreateDiff(w http.ResponseWriter, r *http.Request, ps httprout
 		h.WriteErrorResponse(w, http.StatusInternalServerError, err)
 		return
 	}
+	if err := h.Manager.Validate(repo, req); err != nil {
```

Comment:
This will generate code and therefore when we call `h.DiffCreator.LandDiff()` the diff that gets created will now contain generated code instead of only config files.

I think this `Validate()` call is fine. We just need to update the implementation of `DiffCreater#LandDiff` in our internal codebase to somehow `git reset --hard origin/master -- build/` or something like that.

Reply:
good catch, as synced offline, will add a flag in generate to not alter files in the diff

as discussed, code generate should happen after diff land, just not here :)